src/task3.java

```diff
@@ -0,0 +1,152 @@
+public class task3 {
+    public static void main(String[] args) {
+        // Задание 1.
+        byte beverages = 8;
+        System.out.println("гозировка = " + beverages);
+        short loaf = 4;
+        System.out.println(" батон = " + loaf);
+        int tomatos = 5;
+        System.out.println(" помидоры = " + tomatos);
+        long cheese = 2;
+        System.out.println( " сыр = " + cheese);
+        float appleBox = 5.26F;
+        System.out.println(" яблоки = " + appleBox);
+        double pearsBox = 8.456;
+        System.out.println(" груши = " + pearsBox);
+
+        // Задание 2.
+        byte sweets = 67;
+        System.out.println(sweets);
+        short peapers = -159;
+        System.out.println(peapers);
+        int hauses = 569;
+        System.out.println(hauses);
+        double bicycles = 2.786;
+        System.out.println(bicycles);
+        short peoples = 27897;
+        System.out.println(peoples);
+        long maneysBanks = 987678965549L;
+        System.out.println(maneysBanks);
+        float potatoWeight = 27.12F;
+        System.out.println(potatoWeight);
+
+        // Задание 3.
+        byte ЛюдмилаПавловна = 23;
+        System.out.println("у Людмилы Павловны " + ЛюдмилаПавловна + " ученика");
+        byte АннаСергеевна = 27;
+        System.out.println("У Анны Сергеевны " + АннаСергеевна + " учеников");
+        byte ЕкатеринаАндреевна = 30;
+        System.out.println("у Екатерины Андреевны " + ЕкатеринаАндреевна + " учеников");
+        short sheetsPurchasedTotal = 480;
+        System.out.println(sheetsPurchasedTotal);
+        int totalStudents = ЛюдмилаПавловна + АннаСергеевна + ЕкатеринаАндреевна;
+        System.out.println("всего учеников " + totalStudents);
+        int sheetForEachStudent = sheetsPurchasedTotal / totalStudents;
+        System.out.println("на каждого ученика расчитано " + sheetForEachStudent + " листов бумаги");
+
+        // Задача 4.
+        byte time2 = 16;
+        System.out.println("производит машина за 2 минуты = " + time2 + " штук бутылок");
+        int time20 = time2 * 20 / 2;
+        System.out.println("производит машина за 20 минут = " + time20 + " штук бутылок");
+        int time24 = time2 * 1440 / 2;
+        System.out.println("производит машина за 1440 минут = " + time24 + " штук бутылок");
+        int time4320 = time2 * 4320 / 2;
+        System.out.println("производит машина за 4320 минут = " + time4320 + " штуук бутылок");
+        int time43200 = time2 * 43200 / 2;
+        System.out.println("производит машина за 43 200 минут = " + time43200 + " штук бутылок");
+
+        // Задача 5.
+        byte brownPaint = 4;
+        byte whitePaint = 2;
+        int allPaints = brownPaint + whitePaint;
+        System.out.println("краски нужно на один класс = " + allPaints + " шт");
+        byte paint = 120;
+        byte allPaint = 6;
```

Comment:
откуда число? здесь должно быть
```suggestion
        byte allPaint = brownPaint + whitePaint;
```